src/Uno.UI/Generated/3.0.0.0/Windows.UI.Xaml.Controls.Maps/MapTileSource.cs

```diff
@@ -333,8 +333,8 @@ public  bool AutoPlay
 			typeof(global::Windows.UI.Xaml.Controls.Maps.MapTileSource), 
 			new FrameworkPropertyMetadata(default(global::System.TimeSpan)));
 		#endif
-		#if __ANDROID__ || __IOS__ || NET461 || __WASM__ || __SKIA__ || __NETSTD_REFERENCE__ || __MACOS__
-		[global::Uno.NotImplemented("__ANDROID__", "__IOS__", "NET461", "__WASM__", "__SKIA__", "__NETSTD_REFERENCE__", "__MACOS__")]
+		#if __ANDROID__ || false || false || false || false || false || __MACOS__
+		[global::Uno.NotImplemented("__ANDROID__", "__MACOS__")]
 		public MapTileSource() : base()
```

Comment:
@ebariche I don't quite understand why the generator made this change? `MapTileSource` doesn't seem to be implemented in any platform. I'll have to debug through this tomorrow morning (it's quite late here) if the reason isn't immediately apparent.

Reply:
What was happening is that the generated "DependencyObjectGenerator" code was considered as implementing the constructor. For now, deleting `Uno.UI/obj` before running the generator fixes it. We'll probably need to look at something better than this in future.

Reply:
I'm going to run it on my side and see. Could you squash all the commits ? I want to keep the history clean for future rebasing !

Reply:
Also could you remove => chore: Add SelectAll to PasswordBox and the generated change to PasswordBox.cs

Reply:
@ebariche For the PasswordBox, I think the change is necessary.

Running the generator without manual changes marks `SelectAll` as implemented (well, that's probably a generator bug, it assumed it's implemented because it's inherited from TextBox, and that's another bug because PasswordBox shouldn't inherit from TextBox).

With only a generator run which removes SelectAll, PackageDiff will flag that as a breaking change (because it removes PasswordBox.SelectAll altogether)